sdk/samples/core/mosquitto_rpc_client_sample.c

```diff
@@ -0,0 +1,303 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file
+ * @brief Mosquitto RPC client sample
+ *
+ */
+
+#include <az_log_listener.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <uuid/uuid.h>
+
+#include "rpc_client_pending_commands.h"
+#include <azure/az_core.h>
+#include <azure/core/az_log.h>
+#include <azure/core/az_mqtt5_rpc.h>
+#include <azure/core/az_mqtt5_rpc_client.h>
+
+#ifdef _WIN32
+// Required for Sleep(DWORD)
+#include <Windows.h>
+#else
+// Required for sleep(unsigned int)
+#include <unistd.h>
+#endif
+
+static const az_span cert_path1 = AZ_SPAN_LITERAL_FROM_STR("<path to pem file>");
+static const az_span key_path1 = AZ_SPAN_LITERAL_FROM_STR("<path to key file>");
+static const az_span client_id = AZ_SPAN_LITERAL_FROM_STR("mobile-app");
+static const az_span username = AZ_SPAN_LITERAL_FROM_STR("mobile-app");
```

Comment:
```suggestion
static const az_span username = AZ_SPAN_LITERAL_FROM_STR("mobile-user");
```

Reply:
`mobile-app` keeps parity with the MQTT Application Samples/Concept Examples